controller/iteration.go

```diff
@@ -19,6 +19,12 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+// Following const are keys to be used in Meta Relationship for counters of Work Item
+const (
+	KeyTotalWorkItems  = "total"
+	KeyClosedWorkItems = "closed"
```

Comment:
These two exported constants are used only within black-box test cases, right? If so, I would suggest to move it there.

Reply:
I agree @baijum

Reply:
instead, I refactored other pieces to use these constants, please see here https://github.com/fabric8-services/fabric8-wit/pull/1666/commits/baab87f3adb189d8f4a9be64e0a745748b86e155
Let me know if it is okay. Thanks!

Reply:
Well done @pranavgore09